nengo/tests/test_node.py

```diff
@@ -139,25 +139,31 @@ def test_circular(Simulator, seed):
     assert np.allclose(sim.data[a_p], sim.data[b_p])
 
 
-def test_function_args_error(Simulator):
+def test_outputparam_errors(Simulator):
     with nengo.Network() as model:
+        # type errors
```

Comment:
The last line of this section is a "type" error (i.e. the node output is not an acceptable type, not literally a TypeError, though). But you're right, this is mostly checking that the supported types work fine. Would you prefer no comment? Or a different comment?